test/collectionUtils/core/AllPredicate.test.ts

```diff
@@ -0,0 +1,38 @@
+import AllPredicate from "../../../src/collectionUtils/core/AllPredicate";
+import FalsePredicate from "../../../src/collectionUtils/core/FalsePredicate";
+import TruePredicate from "../../../src/collectionUtils/core/TruePredicate";
+
+describe("AllPredicate", () => {
+  it("should return true when no predicates are specified", () => {
+    const predicate = new AllPredicate<string>();
+    expect(predicate.evaluate("value")).toBeTrue();
+  });
+
+  it("should handle single truthy predicate", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)